src/main/java/synapseawsconsolelogin/Auth.java

```diff
@@ -339,11 +346,17 @@ public void initProperties() {
 			propertyFileName = "global.properties";
 			
 		}
-		
+
+		properties.putAll(loadProperties(propertyFileName));
```

Comment:
from java doc, https://docs.oracle.com/javase/7/docs/api/java/util/Properties.html

```
Because Properties inherits from Hashtable, the put and putAll methods can be applied to a Properties object. Their use is strongly discouraged as they allow the caller to insert entries whose keys or values are not Strings. The setProperty method should be used instead. If the store or save method is called on a "compromised" Properties object that contains a non-String key or value, the call will fail. Similarly, the call to the propertyNames or list method will fail if it is called on a "compromised" Properties object that contains a non-String key.
```

Reply:
Will check.